purchase_mrp_ux/README.rst

```diff
@@ -0,0 +1,71 @@
+.. |company| replace:: ADHOC SA
+
+.. |company_logo| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-logo.png
+   :alt: ADHOC SA
+   :target: https://www.adhoc.com.ar
+
+.. |icon| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-icon.png
+
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===============
+Purchase MRP UX
+===============
+
+This module block the refund picking with products kits.
```

Comment:
Poner:
"
This module blocks the return of purchased products of type kit with the "To Refund" option.
TODO: we should allow this and fix computation of received quantity and compute returned quantity
"